files/bootstrap.sh

```diff
@@ -306,6 +306,8 @@ if [[ "$USE_MAX_PODS" = "true" ]]; then
     fi
 fi
 
+mkdir -p /etc/systemd/system/kubelet.service.d
+
 cat <<EOF > /etc/systemd/system/kubelet.service.d/10-kubelet-args.conf
```

Comment:
Since we cat directly to that folder, it makes sense to ensure the folder exists first.